src/components/Card/Card.mdx

```diff
@@ -208,6 +209,14 @@ This example can be found in the [Turbine homepage](https://nypl-ds-test-app.ver
 
 <Canvas of={CardStories.CardFullClickTurbineExample} />
 
+## Card With Full-Click Functionality and Tooltip
+
+The `Tooltip` relies on pointer events that are overridden by the card's `LinkOverlay`.
```

Comment:
`LinkOverlay` is not mentioned in this doc besides this. Can you say that it's an internal component that is added through the  "full-click" functionality? Just to be clear how/when it's added and why it affects the tooltip.